py-polars/tests/unit/io/test_parquet.py

```diff
@@ -784,3 +785,33 @@ def test_parquet_array_statistics() -> None:
     assert pl.scan_parquet("test.parquet").filter(
         pl.col("a") != [1, 2, 3]
     ).collect().to_dict(as_series=False) == {"a": [[4, 5, 6], [7, 8, 9]], "b": [2, 3]}
+
+
+@pytest.mark.write_disk()
```

Comment:
Can we also mark this as `slow`? Since we're creating some relatively big data here.

```suggestion
@pytest.mark.slow()
@pytest.mark.write_disk()
```